data/mods/Magiclysm/effects/effects.json

```diff
@@ -1452,6 +1452,13 @@
     "rating": "good",
     "base_mods": { "hit_mod": [ 2 ] }
   },
+  {
+    "type": "effect_type",
+    "id": "effect_feral_stormshaper_stormshammer",
```

Comment:
I think you have an extra s in the "stormhammer" here? Unless it's supposed to be storm**s**hammer. Would also mean changing all the IDs in the attacks of course.

Reply:
In this instance, the effect is only referenced by the monsters in attack conditions and isn't ever visible to the player, so as long as the id is consistent the exact id isn't important.

Reply:
Ahh, gotcha! I'll keep that in mind for the future.